index.html

```diff
@@ -4,6 +4,19 @@
 		<meta charset="utf-8">
 		<meta http-equiv="X-UA-Compatible" content="IE=edge">
 		<title>Simulateur de coût d'embauche</title>
+		<style>
+			body {
+				font-family: 'Helvetica Neue', Arial, Helvetica, sans-serif;
+				line-height: 2.5;
```

Comment:
Once again: I believe the increased `line-height` improves readability and should be part of our text-based interface defaults (i.e. included in the widget's styles, not specific to our example inclusion page).